mlrun/api/db/sqldb/db.py

```diff
@@ -1901,11 +1901,16 @@ def _update_db_record_from_object_dict(
             db_object.created = common_object_dict["metadata"].pop(
                 "created", None
             ) or datetime.now(timezone.utc)
+        db_object.publish_time = common_object_dict["metadata"].pop(
+            "publish_time", None
+        )
```

Comment:
No reason to use `pop` and then re-set the publish-time, since we're not modifying it here under any flow. You could just `get` the `publish_time` from the dict and avoid re-setting the value later.

Reply:
if it exist I need to convert it to str(that's the modification), and if it is None I want it to not exist (that's the reason for pop)...